mermithid/processors/TritiumSpectrum/FakeDataGenerator.py

```diff
@@ -347,24 +432,64 @@ def generate_unbinned_data(self, Q_mean, mass, ROIbound, S, B_1kev, nsteps=10**4
             #Generating finely spaced points on a gaussian
             gaussian_rates = gaussian(K_lineshape, [err_from_B, 0])
 
-            ratesS = convolve(ratesS, gaussian_rates, mode='same')
             ratesB = convolve(ratesB, gaussian_rates, mode='same')
-
+            #if self.ins_res_width_bounds==None:
+            ratesS = convolve(ratesS, gaussian_rates, mode='same')
 
         ratesS[ratesS<0.] = 0.
-        ratesB[ratesB<0.] = 0.
-        rate_sumS, rate_sumB = np.sum(ratesS), np.sum(ratesB)
+        rate_sumS = np.sum(ratesS)
         probsS = np.array(ratesS)/rate_sumS
+        """
+        else:
```

Comment:
maybe delete all the commented and no longer use cases for when ratesS was a list.